src/scenes/Lines/scenes/Editor/General/index.js

```diff
@@ -85,8 +77,10 @@ export default ({
 
       <Dropdown
         className="form-section"
-        searchable
-        items={operators.map(o => ({ value: o.id, label: o.name }))}
+        items={[
+          { value: DEFAULT_SELECT_VALUE, label: DEFAULT_SELECT_LABEL },
+          ...operators.map(o => ({ value: o.id, label: o.name }))
```

Comment:
destruct:a `id` och `name`? 😊

```suggestion
          ...operators.map(({id, name}) => ({ value: id, label: name }))
```